f2/src/FinalFeedbackPage.js

```diff
@@ -0,0 +1,59 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { Route } from 'react-router-dom'
+import fetch from 'isomorphic-fetch'
+import { Trans } from '@lingui/macro'
+import { H1 } from './components/header'
+import { TrackPageViews } from './TrackPageViews'
+import { Layout } from './components/layout'
+import { FinalFeedbackForm } from './forms/FinalFeedbackForm'
+import { BackButton } from './components/backbutton'
+import { Stack } from '@chakra-ui/core'
+
+async function postData(url = '', data = {}) {
```

Comment:
We should probably extract this postData code to a util directory - it's pretty similar to the code used to post the form data